format/spec.md

```diff
@@ -178,6 +178,8 @@ A **`list`** is a collection of values with some element type. The element field
 
 A **`map`** is a collection of key-value pairs with a key type and a value type. Both the key field and value field each have an integer id that is unique in the table schema. Map keys are required and map values can be either optional or required. Both map keys and map values may be any type, including nested types.
 
+A **`variant`** is a type to represent semi-structured data. A variant value can store a value of any other type, including any primitive, struct, list or map values. The variant value is encoded in its own binary [encoding](https://github.com/apache/parquet-format/blob/master/VariantEncoding.md). Variant type is added in [v3](#version-3).
```

Comment:
I'm generally paranoid about these things, but should we link to a specific commit in the parquet repository?

Reply:
You are concerned that the link in master may break? Make sense. Let me update that.

Reply:
Updated. Currently I'm linking to the commit which merges the spec. We can update to a released Parquet branch later.